Tests/FluentAssertions.Specs/Xml/XDocumentAssertionSpecs.cs

```diff
@@ -1231,6 +1231,221 @@ public void When_asserting_a_document_has_an_element_with_a_null_xname_it_should
         }
     }
 
+    public class HaveElementWithValue
+    {
+        [Fact]
+        public void Throws_when_null()
+        {
+            // Arrange
+            XDocument element = null;
+
+            // Act
+            Action act = () => element.Should().HaveElementWithValue("child", "b");
+
+            // Assert
+            act.Should().Throw<XunitException>().WithMessage("*child*b*element itself is <null>*");
+        }
+
+        [Fact]
+        public void Has_element_with_specified_value()
+        {
+            // Arrange
+            var element = XDocument.Parse(
+                """
+                <parent>
+                    <child>a</child>
+                    <child>b</child>
+                </parent>
+                """);
+
+            // Act / Assert
+            element.Should().HaveElementWithValue("child", "b");
+        }
+
+        [Fact]
+        public void Throws_when_element_not_found()
```

Comment:
```suggestion
        public void Throws_when_element_is_not_found()
```

Reply:
🔧  Although we prefer short names, they should still be proper English. Same for all the other tests.